business_objects/monitor.py

```diff
@@ -0,0 +1,424 @@
+from typing import Any, List
+from . import general
+from .. import enums
+
+
+def get_all_tasks(project_id: str = None, only_running: bool = False) -> List[Any]:
+    query = f""" 
+    SELECT tasks.*, orga.name organization_name
+    FROM (
+        ({__select_running_information_source_payloads(project_id, only_running)})
+        UNION
```

Comment:
I'm assuming UNION ALL (UNION also checks if duplications exist which is unnecessary here since there is no overlap)

Reply:
Done